data/brands/tourism/hotel.json

```
@@ -61,7 +61,41 @@
     {
       "displayName": "AC Hotel",
       "id": "achotel-779ccb",
-      "locationSet": {"include": ["001"]},
+      "locationSet": {
+        "include": [
+          "at",
+          "au",
+          "ca-qc.geojson", 
```

I know it might be a little confusing that some of these .geojsons are their full name while others are ISO 3166-2. A list of .geojson features can be found in the [features directory](https://github.com/osmlab/name-suggestion-index/tree/main/features)
```suggestion
          "quebec.geojson", 
```